### ISSUE TITLE
Does not auto-save by default, can close edited document without any warning about saving it first

### ISSUE THREAD

**Issue by zilexa** [ONLYOFFICE/Docker-DocumentServer#345]
**Do you want to request a *feature* or report a *bug*?**
Bug as it has consequences for commonly expected behaviour. 

**What is the current behavior?**
When I open a document, I can edit it and close my screen. OnlyOffice does not warn the changes are not saved. Other web services do give a warning before closing a tab. 

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
1. Open a file from nextcloud for example in OnlyOffice Documentserver. 
2. Edit the document. 
3. Close OnlyOffice. 
4. Open the document again, notice the changes are not present. 
5. Do step 1 and 2, this time save the document manually. then close it. 
6. Open the document, notice changes are saved. 

**What is the expected behavior?**
1. At least a warning "do you want to save changes". 
2. auto-save should be on by default, or that should be an option via Docker env variables.

**Did this work in previous versions of DocumentServer?**
I don't know. 

**DocumentServer Docker tag:**
image: onlyoffice/documentserver

**Host Operating System:**
Ubuntu Budgie 20.10

**Comment by ShockwaveNN** [comment]
@zilexa Hi, could you try to check same scenario using integrated test example (you can enable it temporarily via command descrbed in http://doc-server-url/welcome) and check if problem with save message appears for clean DocumentServer?

**Comment by zilexa** [comment]
Interesting! With the example, auto-save is enabled by default!
Only when I open something from my fileserver it is off. 
Also the layout seems different:

Going to office.mydomain.com and creating a new file from the example: 
![2021-03-24 14_57_07-Window](https://user-images.githubusercontent.com/3430004/112322573-50f0f200-8cb1-11eb-964a-040ad4f1316b.png)

Opening an existing doc from my fileserver, perhaps since this is opened from the fileserver domain, it looks different?:
![2021-03-24 14_57_35-Window](https://user-images.githubusercontent.com/3430004/112322684-68c87600-8cb1-11eb-9ed1-b6aa5e8672da.png)

**Comment by ShockwaveNN** [comment]
> With the example, auto-save is enabled by default!

It's enabled by default but maybe you're disabled it and saved this setting, it's stored in browser localstorage

**Comment by ShockwaveNN** [comment]
But anyway even with autosave disabled - there should be message about losing data, you can see it via test example

**Comment by zilexa** [comment]
OK tested: 
1. In a private window, I do get a popup if I try to close a newly created or existing doc (opened via my fileserver) or spreadsheet with unsaved changes. So that's good. 
2. In a private window, autosave is off by default for newly created and existing doc and spreadsheets. 
3. When I doubleclick a file in my fileserver, a new tab isn't opened, Onlyoffice is shown embedded in a "popup" within the webpage. I can edit document, close it, no popup will appear, losing changes. When I right-click and open in OnlyOffice (this triggers a new tab), the behaviour is as mentioned in 1. and 2. 

So auto-save is still off by default, except for the test example. Also, the UI looks very different. Any reason why?

**Comment by ShockwaveNN** [comment]
> So auto-save is still off by default, except for the test example. Also, the UI looks very different. Any reason why?

On your screenshot there is only difference in top tab - we changing it for usage in nextcloud, but I cannot see any other differences

**Comment by zilexa** [comment]
> 
> 
> > So auto-save is still off by default, except for the test example. Also, the UI looks very different. Any reason why?
> 
> On your screenshot there is only difference in top tab - we changing it for usage in nextcloud, but I cannot see any other differences

Ok that explains. 
BTW I notice auto-save is disabled again: each time I open a document, I have to manually enable it. 
It seems my fileserver is using some sort of setting to open files with basic/default settings. Things like auto-save, but also standard language (always US English, I would prefer Dutch) are always reset when opening (the same) document.

**Comment by ShockwaveNN** [comment]
@zilexa So, to be clear

There is no problems in incognito mode while you're using DocumentServer test example
There is problems with save while you're using your nextcloud instance

If this is right I've moving this issue to another repo, since my speciality is not in integration with other services, but in clean DocumentServer installation and there may be trouble on nextcloud side

**Comment by zilexa** [comment]
Yes you are correct, I also do not believe the issue is clean OnlyOffice as the example proves auto-save should be on. It has to do with the integration.
